ros2pkg/ros2pkg/api/create.py

```diff
@@ -112,6 +112,10 @@ def create_package_environment(package, destination_directory):
     if package.get_build_type() == 'ament_python':
         print('creating source folder')
         source_directory = _create_folder(package.name, package_directory)
+    if package.get_build_type() == 'ament_cmake_python':
```

Comment:
There are currently no instances of ROS 2 packages where this would be true.  That is, `package.get_build_type()` will never return `ament_cmake_python`.

That said, it would still be worthwhile to have some way to create a package template for a Python package using CMake.  However, it will have to be done a different way because it will use a `build_type` of `ament_cmake`.  Maybe we can add in a new command-line argument that tells it to generate Python installation stuff in the CMakeLists.txt?